docs/reference/logging/index.md

```diff
@@ -25,6 +25,16 @@ container's logging driver. The following options are supported:
 
 The `docker logs`command is available only for the `json-file` logging driver.  
 
+### The json-file options
+
+The following logging options are supported for the `json-file` logging driver:
```

Comment:
@thaJeztah the description written by @moxiegirl was my previous description. It seems to be a little ambiguous. I felt like this was clearer. What do you think?